test/functional/wallet_labels.py

```diff
@@ -25,6 +25,40 @@ def set_test_params(self):
     def skip_test_if_missing_module(self):
         self.skip_if_no_wallet()
 
+    def invalid_label_name_test(self):
+        node = self.nodes[0]
+        address = node.getnewaddress()
+        pubkey = node.getaddressinfo(address)['pubkey']
+        rpc_calls = [
+            [node.getnewaddress],
+            [node.setlabel, address],
+            [node.getaddressesbylabel],
+            [node.importpubkey, pubkey],
+            [node.addmultisigaddress, 1, [pubkey]],
+            [node.getreceivedbylabel],
+        ]
+        if self.options.descriptors:
+            response = node.importdescriptors([{
+                'desc': f'pkh({pubkey})',
+                'label': '*',
+                'timestamp': 'now',
+            }])
+        else:
+            rpc_calls.append([node.importprivkey, node.dumpprivkey(address), "*"])
```

Comment:
Since the asterisk is added in the for-loop below, there is no need to add it here. I'd suggest to also add test coverage for `importaddress` here:
```suggestion
            rpc_calls.extend([
                [node.importprivkey, node.dumpprivkey(address)],
                [node.importaddress, address],
            ])
```

Reply:
Thanks done